app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
delete empty file (not gonna comment on the rest of the coffeescript ones but they should all go)